server/backends/disk_cache/disk_cache.go

```diff
@@ -50,6 +51,9 @@ const (
 	// check the cache size.
 	janitorCheckPeriod = 100 * time.Millisecond
 
+	// refreshMetricsPeriod is how often we update polled metrics.
+	refreshMetricsPeriod = 5 * time.Second
```

Comment:
wdyt about a minute instead?

Reply:
All of the metrics are cheap so I don't see the harm of exporting it more frequently.